recipes/htop/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "htop" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hishamhm/htop.git
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - automake
+    - gcc
+    - libtool
+    - ncurses
```

Comment:
Please pin to `5.9*` to match our [global pinnings]( https://github.com/conda-forge/conda-forge.github.io/blob/70036d169783f7d6844bfc1ff1b360dba5adfa9f/scripts/pin_the_slow_way.py#L45 ).